packages/control-property-editor-common/src/api.ts

```diff
@@ -139,6 +139,7 @@ export interface SavedPropertyChange<T extends PropertyValue = PropertyValue> ex
     kind: 'valid';
     fileName: string;
     timestamp: number;
+    file: object;
```

Comment:
The change object is needed for `sap/ui/fl/FlexObjectFactory` in order to create an instance of a Flex Object, later used to get the Visualization Info. I see one other way, but it would require a fetch from the workspace every time a change in the Outline is done. This will probably have an effect on the performance.

Reply:
I had a discussion with @nikmace and it seems I misunderstood your point. I made a change, where the file content is kept in a local variable and never passed to the UI.
Please check commit: 9931f056702a0c6049bfcef2d63eb5a4c0114bfc